wordpress-mobile/react-native-aztec
# Changes to integrate with WPiOS app.

This PR introduces some changes to integrate with the WPiOS app.

I opted for temporarily remove images and attachment handling in favour of a simpler Bridge implementation. This is to avoid a direct dependency between `Aztec` and `RNReactNativeGutenbergBridge`.

We might find out that this dependency is unavoidable, but for simplicity sake on this first step, I'd prefer to work without it. I plan to reimplement or clean up the commented code as soon as we decide on the path to follow with the general integration.

This PR also adds a `podspec` file, but is not really being used yet. I plan to use it in the near future and I'd like to keep it here for now. We can remove it later if we end up not using it at all.

We need to merge this before merging the main [iOS bridge implementation](https://github.com/wordpress-mobile/gutenberg-mobile/pull/195).

To test:
- Build and run the [iOS bridge implementation](https://github.com/wordpress-mobile/gutenberg-mobile/pull/195) branch.
- Check that there are no errors.
- We expect a crash if the `Paragraph` block contains attachments.
